### ISSUE TITLE
Gradient has outdated direction syntax. New syntax is like "to left" instead of "right"

### ISSUE THREAD

**Issue by karlwilbur** [samcollins/css-social-buttons#90]
```
autoprefixer: /home/karl/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/zocial-rails-b265bd304eac/vendor/assets/stylesheets/zocial.css:122:2: Gradient has outdated direction syntax. New syntax is like "to left" instead of "right".
autoprefixer: /home/karl/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/zocial-rails-b265bd304eac/vendor/assets/stylesheets/zocial.css:206:2: Gradient has outdated direction syntax. New syntax is like "to left" instead of "right".
autoprefixer: /home/karl/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/zocial-rails-b265bd304eac/vendor/assets/stylesheets/zocial.css:236:2: Gradient has outdated direction syntax. New syntax is like "to left" instead of "right".
```

Proper syntax refs:
https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient
http://www.w3.org/TR/css3-images/#linear-gradients

**Comment by zimbatm** [comment]
Fixed by #91 right ?

**Comment by karlwilbur** [comment]
Yes. Thanks.
